mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/TransactionalMessageMapper.java

```diff
@@ -132,7 +132,9 @@ public MessageMetaData copy(final Mailbox mailbox, final MailboxMessage original
 
     @Override
     public MessageMetaData move(Mailbox mailbox, MailboxMessage original) throws MailboxException {
-        return messageMapper.move(mailbox, original);
+        MessageMetaData data = messageMapper.execute(
```

Comment:
We could return directly?

Reply:
I applied the same pattern as in the other methods (add, copy, etc.).
However, I fixed this (and the other methods) in commit [3700eab](https://github.com/linagora/james-project/pull/1850/commits/3700eabab49f92284ddc1b0f8297fc6d9e44a88c).